### ISSUE TITLE
Form controls accessibility items

### ISSUE THREAD

**Issue by teenwolfblitzer** [Kajabi/sage-archived#259]
## Description
Some general housekeeping items to help out with accessibility.

The following items are not associated with their item/control, and may require conditional `aria-describedby` assignment via JS. `aria-labelledby` does not seem to apply here since the input already uses a corresponding `label`:
- [ ] Message text on input fields (`sage-input__message`)
- [ ] Message text on switches (`sage-switch__message`)
- [ ] Message text on selects (`sage-select__message`)

Additional items to address:
- [ ] Add `legend` for any `fieldset` groups
- [ ] Possible need for `role="group"` to reinforce relationship of `.sage-input-group` elements

## Related

- n/a
